app/src/main/res/values/strings.xml

```diff
@@ -28,7 +28,7 @@
     <string name="password">Password</string>
     <string name="manage_events">Manage Events</string>
     <string name="ticket_issues">Ticket Issues</string>
-    <string name="ticket_issues_url">support.eventyay.com</string>
+    <string name="ticket_issues_url">http://support.eventyay.com</string>
```

Comment:
Non translatable things are not recommended in strings.xml

Either move or set translatable = false